src/tensors/cpu/intgemm_interface.h

```diff
@@ -578,10 +613,9 @@ static inline Expr affine(Expr a, Expr b, Expr bias, bool transA, bool transB, f
   if (isIntgemm(bElementType)) {
     //This is the case where we already run SelectColumnB or we loaded a prepacked model.
     //We ignore a transpose argument here, because we do not support it.
```

Comment:
We can remove the comment `//We ignore a transpose argument here, because we do not support it.`

Reply:
Yeah that should be deleted.

Reply:
@XapaJIaMnu Could you just delete this comment? It is good to merge afterwards 👍🏾